site/content/integrate/apps/deployment-aws/deploy-to-aws/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Deployment (Admins)"
+heading: "Deploying AWS Serverless Apps (For Admins)"
+description: "TODO"
+weight: 40
+---
+
+Admins of the on-prem Mattermost instance must prepare both AWS and Mattermost before provisioning and deploying an app.
+
+Note that apps are not supported in the Mattermost Cloud - they are only used for developer testing convenience.
```

Comment:
" apps are not supported' - apps are supported in MM cloud (previous paragraph0, it is just these types of apps that are not, right?

Reply:
Fixed in d3c105a5